app/validators/price_validator.rb

```diff
@@ -0,0 +1,9 @@
+class PriceValidator < ActiveModel::Validator
+  def validate(record)
+    price_value = record.send(:price)
```

Comment:
why have you used send here? what's the nee d of this condition `price_value && discount_value`